### PR TITLE
remove big deps

### PR DESCRIPTION
**Pull request by purge**
Built sqwidget is now 25k with everything included.

### CODE REVIEW

**Comment by adhipg** [comment]
Should we also do a wrapper since we are using both `qwery` and `bonzo` to write `jQuery` like code?

https://github.com/ded/bonzo#use-with-a-selector-engine

File: src/component/core.coffee [hunk 1]
```diff
@@ -1,16 +1,19 @@
-define ['underscore', 'backbone'], (_, Backbone) ->
+define ['heir', 'bonzo', 'EventEmitter'], (heir, bonzo, EventEmitter) ->
 
   class SqwidgetCore
     constructor: () ->
-      _.extend(@, Backbone.Events)
+
+      #sqwidget level event bus
+      @ee = new EventEmitter()
```
Comment (adhipg) [inline src/component/core.coffee:7]: How will this work for inter-widget communication?

Comment (purge) [inline src/component/core.coffee:7]: core is only initialised once, widgets are registered against this instance.

Comment (adhipg) [inline src/component/core.coffee:7]: Right.

**Pull request merged**